share/functions/help.fish

```diff
@@ -100,20 +100,22 @@ function help --description 'Show help for the fish shell'
             # Escaped quotes are necessary to work with spaces in the path
             # when the command is finally eval'd.
             set fish_browser cygstart \"$fish_browser\"
+        else
+            set need_trampoline 1
         end
     end
 
     switch "$fish_help_item"
         case "."
             set fish_help_page "cmds/source.html"
         case globbing
-            set fish_help_page "index.html\#expand"
+            set fish_help_page "index.html#expand"
```

Comment:
Yep, that looks good. The ad-hoc hash escaping is a leftover from when the url was opened with Mac `open` which discards anchors.